routes/user.js

```diff
@@ -38,8 +35,17 @@ router.get('/users/:id', async (req, res) => {
 router.delete('/users/:id', async (req, res) => {
   try {
     const _id = req.params.id;
-    const user = await UserModel.delete(_id);
-    res.status(200).send(user);
+    const user = await models['User'].findByPk(_id);
+    if (user) {
+      await models['User'].destroy({
+        where: {
+          id: _id,
+        },
+      });
+      res.status(200).send(user);
+    } else {
+      res.status(404).send(e);
+    }
   } catch (e) {
     res.status(404).send(e);
```

Comment:
also it's handful to write `console.error('delete user', e)` in each such case. Thus during debugging we will easily and quickly what route the error comes from